continuous-delivery/build-wheels-osx.sh

```diff
@@ -2,11 +2,13 @@
 #before running this, you'll need cmake3 and a compiler. These python versions are just
 #using the default python installers from python.org. Each version needs updated pip, wheel, and setuptools
 set -e
-python setup.py bdist_wheel
+export MACOSX_DEPLOYMENT_TARGET="10.9"
+/Library/Frameworks/Python.framework/Versions/2.7/bin/python setup.py bdist_wheel
 #note python 3.3 on OSX is a no-go, there's no installer. If you can build python 3.3 from source
 # you can build the wheel, easy-peasy
-/Library/Frameworks/Python.framework/Versions/3.4/bin/python3 setup.py bdist_wheel
+#/Library/Frameworks/Python.framework/Versions/3.4/bin/python3 setup.py bdist_wheel
```

Comment:
No more 3.4? Is this because it's deprecated? Should we just delete the line then?

Reply:
there's not even an installer for it.